### DISCUSSION TITLE
DWpicker null duplicate errors

### DISCUSSION THREAD

**Discussion by Marccss** [community/community#87623] [Programming Help]
### Select Topic Area

Question

### Body

I have created a picker for a face in Maya.
When I select all of the controls on the picker I get an error telling me there are duplicates of some of the nodes/ controls.
I have deleted the listed nodes and recreated the controls but get the same error.
I have deleted the json file and resaved the file with a different name and get the same error.
I have attached a picture of the error with the listed nodes.

Any help would be appreciated.

<img width="257" alt="Fox_picker_error" src="https://github.com/community/community/assets/34040601/b1f520c4-5122-4b57-95fd-acce2bdf3967">

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Marccss ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
